File: frontend/src/app/dto/comment/comment.data.dto.ts
export class CommentDataDto{
    private _id! : number
    private _comments! : string
    private _ticketId! : number
    private _userId! : number
    private _fullname! : string
    private _createdAt! : string
	private _ver! : number
    private _fileId! : number[]
    private _photoId! : number

    public get photoId(): number {
        return this.photoId;
    }

    public set photoId(photoId: number) {
        this._photoId = photoId;
    }

    public get fileId(): number[] {
        return this._fileId;
    }

    public set fileId(fileId: number[]) {
        this._fileId = fileId;
    }

    public get comments(): string {
        return this._comments;
    }

    public set comments(comments: string) {
        this._comments = comments;
    }

    public get ticketId(): number {
        return this._ticketId;
    }

    public set ticketId(ticketId: number) {
        this._ticketId = ticketId;
    }

    public get userId(): number {
        return this._userId;
    }

    public set userId(userId: number) {
        this._userId = userId;
    }

    public get fullname(): string {
        return this._fullname;
    }

    public set fullname(fullname: string) {
        this._fullname = fullname;
    }

    public get createdAt(): string {
        return this._createdAt;
    }

    public set createdAt(createdAt: string) {
        this._createdAt = createdAt;
    }

    public get id(): number {
        return this._id;
    }

    public set id(id: number) {
        this._id = id;
    }

    public get ver(): number {
        return this._ver;
    }

    public set ver(ver: number) {
        this._ver = ver;
    }
    
}

File: backend/src/main/java/com/lawencon/ticketms/dto/comment/CommentDataDto.java
package com.lawencon.ticketms.dto.comment;

import java.time.LocalDateTime;
import java.util.List;

public class CommentDataDto {
	private Long id;
	private String comments;
	private Long ticketId;
	private Long userId;
	private String fullname;
	private Long photoId;
	private LocalDateTime createdAt;
	private Integer ver;
	private List<Long> fileId;
	public Long getPhotoId() {
		return photoId;
	}
	public void setPhotoId(Long photoId) {
		this.photoId = photoId;
	}
	public List<Long> getFileId() {
		return fileId;
	}
	public void setFileId(List<Long> fileId) {
		this.fileId = fileId;
	}
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public String getComments() {
		return comments;
	}
	public void setComments(String comments) {
		this.comments = comments;
	}
	public Long getTicketId() {
		return ticketId;
	}
	public void setTicketId(Long ticketId) {
		this.ticketId = ticketId;
	}
	public Long getUserId() {
		return userId;
	}
	public void setUserId(Long userId) {
		this.userId = userId;
	}
	public String getFullname() {
		return fullname;
	}
	public void setFullname(String fullname) {
		this.fullname = fullname;
	}
	public LocalDateTime getCreatedAt() {
		return createdAt;
	}
	public void setCreatedAt(LocalDateTime createdAt) {
		this.createdAt = createdAt;
	}
	public Integer getVer() {
		return ver;
	}
	public void setVer(Integer ver) {
		this.ver = ver;
	}
}
